csharp/NEP17/NEP17.cs

```diff
@@ -10,6 +10,8 @@ namespace Neo.SmartContract.Examples
     [ManifestExtra("Description", "This is a NEP17 example")]
     [SupportedStandards("NEP-17")]
     [ContractPermission("*", "onNEP17Payment")]
+    [ContractPermission("0xfffdc93764dbaddd97c48f252a53ea4643faa3fd", "update")]
```

Comment:
`*`?

Reply:
For static calls, the compiler should know which permissions to add.

Reply:
I will change the compiler

Reply:
OK, I revert this commit